### ISSUE TITLE
isToday(new UTCDate()) => false

### ISSUE THREAD

**Issue by tilowestermann** [date-fns/date-fns#3730]
There is an issue with the [isToday function](https://date-fns.org/v3.4.0/docs/isToday)

  ## The problem

  ### 💻 Code demonstrating the problem

  

  ```js
  import { UTCDate } from '@date-fns/utc';
  import { isToday } from 'date-fns';

  isToday(new UTCDate());
  //=> false
  ```

Demo: https://stackblitz.com/edit/stackblitz-starters-bdluhl?file=src%2Fmain.ts

  ### 🙁 Actual behavior

`new UTCDate()` is not today

  ### 🙂 Expected behavior

`new UTCDate()` is today

  ## Debug information

  - date-fns version: 3.4.0 (@date-fns/utc: 1.2.0)
  - Browser/Node.js version: Edge
  - Your timezone: UTC+1
  - Your current time: 15:00

**Comment by kossnocorp** [comment]
@fturmel, you're right about that. Thank you for pointing the way. I will review it and ship a fix.

**Comment by kossnocorp** [comment]
Fixed via https://github.com/date-fns/date-fns/commit/b1a5eb5d76edcae59be2d502ac66c255198dbdeb

**Comment by kossnocorp** [comment]
Shipped with [`date-fns@3.5.0`](https://github.com/date-fns/date-fns/releases/tag/v3.5.0)
